Makefile

```diff
@@ -51,6 +51,17 @@ man:
 shell: build-container
 	$(DOCKER_RUN_DOCKER) bash
 
+test-style: test-fmt test-vet test-lint
+
+test-fmt:
+	test -z "$$(gofmt -s -l . | grep -v ^vendor/ | tee /dev/stderr)"
```

Comment:
While I like these, isn't it better to call the corresponding shell files in the container? Like for test-integration?